composer.json

```diff
@@ -34,7 +34,7 @@
         "php": "8.1 - 8.4",
         "ext-pdo": "*",
         "ext-pdo_pgsql": "*",
-        "yiisoft/db": "dev-master"
+        "yiisoft/db": "1.3.*"
```

Comment:
This will not work. The package `yiisoft/db-pgsql: master` requires new methods which represented in `yiisoft/db: master`